Update xamarin-notification-hubs-push-notifications-android-gcm.md

jgold6 opened the pull request:
Add section to register the receivers in the AndroidManifest.xml file. This needs to be done so the app registers with FCM and Azure Notification hubs, without it the notifications do not get received.

azurecla commented:
Hi __@xt0rted__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ggailey777 commented:
@xt0rted thank for stopping on those weird curly quotes. We appreciate your help improving Azure docs.

xt0rted commented:
@ggailey777 the [nav menu on the left](https://azure.microsoft.com/en-us/documentation/articles/websites-dotnet-webjobs-sdk-storage-queues-how-to/) doesn't work in IE 11 (stopped about a month ago). Sometimes it will if you navigate directly to a page, but most of the time it doesn't.

jgold6 committed:
Update xamarin-notification-hubs-push-notifications-android-gcm.md

Add section to register the receivers in the AndroidManifest.xml file. This needs to be done so the app registers with FCM and Azure Notification hubs, without it the notifications do not get received.
